config/routes.rb

```diff
@@ -12,4 +12,14 @@
   # Defines the root path route ("/")
   root "users#show"
   # root "articles#index"
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :posts, format: :json do
+          resources :comments, format: :json
+        end
+      end
+    end
+  end
```

Comment:
- [ ] Good job so far 👏 👏 It looks like you have not added yet the **API endpoints for authentication of both registration and login**

- Login endpoint - POST: `/login`
- Registration endpoint - POST: `/register` or `/signup`